### DISCUSSION TITLE
getting started with gsoc 22

### DISCUSSION THREAD

**Discussion by Sharmalm** [pytorch/ignite#2443] [General]
Hello everyone,
I am Lalit mohan sharma, I am new to open source and want to contribute in GSoC 22.
I have a knowlege about python and data science.
Could you please guide me how to get started.

**Comment by vfdev-5** [comment]
@Lalitmohansharma1 thanks for your interest! We haven't yet published ideas for GSoC2022. You can check out [GSoC2021 ideas](https://github.com/pytorch/ignite/wiki/GSoC-2021-project) to get started.
